### DISCUSSION TITLE
I can,t open it with idle,because rocket char

### DISCUSSION THREAD

**Discussion by flashxin** [ultralytics/yolov5#5563] [Ideas]
i want open  xxx.py files,but nothing on my screen.i have to open it with txt, and delete all rocket🚀 char。 everything is ok，i dont know why。so，i write here to help people who always use idle to edit python code，good luck。

**Comment by glenn-jocher** [comment]
@flashxin for working with python I would recommend PyCharm, VSCode or Spyder. All are great IDEs. I use PyCharm myself.

**Reply by flashxin** [reply]
Thank you！I am  downloading  PyCharm。I'm so excited to hear from you that I even want to show off in front of my  all friends。
